app/validators/email_format_validator.rb

```diff
@@ -0,0 +1,16 @@
+class EmailFormatValidator < ActiveModel::EachValidator
+
+  EMAIL_PATTERN = /\A[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,6}\z/i
```

Comment:
I still wonder if we need to be this strict with email validation, but I like that we have it encapsulated within a validator now.